docs/api/en/core/InterleavedBuffer.html

```diff
@@ -75,6 +80,18 @@ <h3>[property:Usage usage]</h3>
 
 		<h2>Methods</h2>
 
+		<h3>[method:this addUpdateRange]( [param:Number start], [param:Number count] )</h3>
+		<p>
+			Adds a range of data in the data array to be updated on the GPU. Adds an
+			object describing the range to the [page:BufferAttribute.updateRanges updateRanges]
```

Comment:
This should be `InterleavedBuffer` instead of `BufferAttribute` .

The same issue appears in `clearUpdateRanges()`, too.

Reply:
Thanks - just fixed it!